archive-node.md

```diff
@@ -57,10 +58,78 @@ curl https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/master/confi
 curl https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/master/genesis_mainnet.yaml > $IOTEX_HOME/etc/genesis.yaml
 curl https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/master/trie.db.patch > $IOTEX_HOME/data/trie.db.patch
 curl https://storage.googleapis.com/blockchain-golden/poll.mainnet.db > $IOTEX_HOME/data/poll.db
-
-#download snapshot and unzip data
-// TODO: add download link
 ```
+## <a name="download"/>Download Data
```

Comment:
don't use a tag for a whole paragraph

Reply:
oh. you forgot to close the a tag.

Reply:
this is how we set a clickable text/tag on the page, so you can click the text and navigate to the the corresponding paragraph. Same as `README.md`
or what is the usage you recommend?